xbmc/settings/AdvancedSettings.cpp

```diff
@@ -767,6 +767,18 @@ void CAdvancedSettings::ParseSettingsFile(const std::string &file)
         XMLUtils::GetFloat(pRefreshVideoLatency, "hdrextradelay", videolatency.hdrextradelay,
                            -600.0f, 600.0f);
 
+        const char* szWidth = pRefreshVideoLatency->Attribute("width");
```

Comment:
i think can be appropriate use `QueryUnsignedAttribute` without the use of stoul that can raise exceptions